addons/calendar/tests/test_event_recurrence.py

```diff
@@ -696,6 +696,49 @@ def test_shift_future_multiple_weekdays(self):
         self.assertEqual(event.recurrence_id.count, 2)
 
 
+class TestUpdateMultiDayWeeklyDayNotIncludedRecurrentEvents(TestRecurrentEvents):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.event = cls.env['calendar.event'].create({
+            'name': 'Recurrent Event',
+            'start': datetime(2019, 10, 22, 1, 0),
+            'stop': datetime(2019, 10, 22, 2, 0),
+            'recurrency': True,
+            'rrule_type': 'weekly',
+            'tue': False,
+            'wed': True,
+            'fri': True,
+            'interval': 1,
+            'count': 3,
+            'event_tz': 'Etc/GMT-4',
+        })
+        cls.recurrence = cls.env['calendar.recurrence'].search([])
+        cls.events = cls.recurrence.calendar_event_ids.sorted('start')
+        # Tuesday datetime(2019, 10, 22, 1, 0)
+        # Wednesday datetime(2019, 10, 23, 1, 0)
+        # Friday datetime(2019, 10, 25, 1, 0)
+        # Wednesday datetime(2019, 10, 30, 1, 0)
```

Comment:
Why are you adding a new test class ? The setup will be heavy for just one single test. You can do everything in the current class. Your test coould be in the class TestUpdateRecurrentEvents. Thanks :-)